TTS is broken

I'm on Android 12. TTS works perfectly on my older Android 6 device.

Version: 8.5.45-fdroid

Details: in the 'TTS Settings' tab, 'Voice' only shows the trashy TTS engine bundled with my system, even though I have already installed Google TTS and made it default. So I disabled the system TTS package with `adb`. But now 'Voice' only displays `--- ---`. When I tap the 'play' icon, I am asked to install the Google Speech Service app, which I've installed.

i am not able to generate fdroid apk.

if i use java11 i get following error
```
Exception while marshalling /opt/android-sdk/build-tools/31.0.0/package.xml. Probably the SDK is read-only
Exception while marshalling /opt/android-sdk/tools/package.xml. Probably the SDK is read-only
Exception while marshalling /opt/android-sdk/build-tools/31.0.0/package.xml. Probably the SDK is read-only
Exception while marshalling /opt/android-sdk/tools/package.xml. Probably the SDK is read-only
Checking the license for package Android SDK Platform 31 in /opt/android-sdk/licenses
License for package Android SDK Platform 31 accepted.
Preparing "Install Android SDK Platform 31 (revision: 1)".
Warning: Failed to read or create install properties file.
```
and `sdkmanager --licenses`
```
Exception in thread "main" java.lang.NoClassDefFoundError: javax/xml/bind/annotation/XmlSchema
        at com.android.repository.api.SchemaModule$SchemaModuleVersion.<init>(SchemaModule.java:156)
        at com.android.repository.api.SchemaModule.<init>(SchemaModule.java:75)
        at com.android.sdklib.repository.AndroidSdkHandler.<clinit>(AndroidSdkHandler.java:81)
        at com.android.sdklib.tool.sdkmanager.SdkManagerCli.main(SdkManagerCli.java:73)
        at com.android.sdklib.tool.sdkmanager.SdkManagerCli.main(SdkManagerCli.java:48)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.annotation.XmlSchema
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        ... 5 more
1 sak@godbless in temp/j
```

if i use java8 (or 1.8. reminds me why i quit java)

```
* Where:
Build file '/home/sak/temp/java/LibreraReader/app/build.gradle' line: 1

* What went wrong:
A problem occurred evaluating project ':app'.
> Failed to apply plugin 'com.android.internal.application'.
   > Android Gradle plugin requires Java 11 to run. You are currently using Java 1.8.
     Your current JDK is located in  /usr/lib/jvm/java-8-openjdk/jre
     You can try some of the following options:
       - changing the IDE settings.
       - changing the JAVA_HOME environment variable.
       - changing `org.gradle.java.home` in `gradle.properties`.
  ```

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -10,7 +10,6 @@
 
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />
 
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
@@ -33,6 +32,9 @@
         </intent>
         <intent>
             <action android:name="android.intent.action.SEARCH" />
+        </intent>
+        <intent>
+          <action android:name="android.intent.action.TTS_SERVICE" />
         </intent>
     </queries>
 

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -10,6 +10,7 @@
 
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />
 
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />

